src/internal/commands.rs

```diff
@@ -51,7 +51,7 @@ impl ShellCommand {
     }
 
     pub fn sudo() -> Self {
-        Self::new("sudo")
+        Self::new(&config::read().bin.sudo.unwrap_or_default())
```

Comment:
Change this to `unwrap_or_else("sudo")` or we'll end up with an empty string

Reply:
what is the point of impl Default then?

Reply:
Currently it's just used to return a default config when the condig file doesn't exist.
This works because we're calling `default` on the config struct.
What you're doing is calling `default` on a string which always returns an empty string.

Reply:
that is not the behaviour I expected at all

Reply:
At some point I'll reimplent the config in a way that allows fallbacks from the Default impl of the config the way you expected it to work.